src/Microsoft.Azure.WebJobs.Host.Storage/BlobStorageConcurrencyStatusRepository.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace Microsoft.Azure.WebJobs.Host
 {
-    internal class BlobStorageConcurrencyStatusRepository : IConcurrencyStatusRepository
+    public class BlobStorageConcurrencyStatusRepository : IConcurrencyStatusRepository
```

Comment:
Rather than making this implementation public, scale hosts should register this implementation using an appropriate host builder method. Currently, AddAzureStorageCoreServices is called to register this implementation. If we can't use that method, we could introduce a more targeted one, the goal being to keep this type internal.